src/index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+// Register means every single file will get processed by babel that we import!
+import 'babel/register'
```

Comment:
this is not quite right because this file actually get's compiled... it's the nested dependencies which it loads that end up using "register"